packages/preview/esotefy/1.0.0/typst.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "esotefy"
+version = "1.0.0"
+entrypoint = "index.typ"
+authors = ["Thumus"]
+license = "MIT"
+description = "A brainfuck implementation in pure Typst"
+repository = "git@github.com:Thumuss/brainfuck.git"
+homepage = "https://github.com/Thumuss/brainfuck"
+keywords = [
+    "brainfuck",
+    "esoteric",
+    "langage",
+]
+compiler = "0.7.0"
+exclude = ["example.typ", "example.pdf", ".gitignore", "LICENSE"]
```

```suggestion
exclude = ["example.typ", "example.pdf", ".gitignore"]
```
The LICENSE should be distributed with the package.